src/hotspot/share/classfile/stackMapTable.cpp

```diff
@@ -339,27 +350,28 @@ StackMapFrame* StackMapReader::next(
         locals = nullptr;
       }
     } else {
-      offset = pre_frame->offset() + offset_delta + 1;
+      offset = _pre_frame->offset() + offset_delta + 1;
     }
     frame = new StackMapFrame(
-      offset, flags, new_length, 0, max_locals, max_stack,
+      offset, flags, new_length, 0, _max_locals, _max_stack,
       locals, nullptr, _verifier);
-    if (first && locals != nullptr) {
-      frame->copy_locals(pre_frame);
+    if (_first && locals != nullptr) {
+      frame->copy_locals(_pre_frame);
     }
+    _first = false;
     return frame;
   } else if (frame_type < SAME_EXTENDED + 4) {
     // append_frame
     int appends = frame_type - SAME_EXTENDED;
-    int real_length = pre_frame->locals_size();
+    int real_length = _pre_frame->locals_size();
     int new_length = real_length + appends*2;
     locals = NEW_RESOURCE_ARRAY_IN_THREAD(THREAD, VerificationType, new_length);
-    VerificationType* pre_locals = pre_frame->locals();
+    VerificationType* pre_locals = _pre_frame->locals();
     int i;
-    for (i=0; i<pre_frame->locals_size(); i++) {
+    for (i=0; i<_pre_frame->locals_size(); i++) {
```

Comment:
This needs spaces too since you touched it.  Can "int i" be in scope of the for loop too?